modules/ossm-migrating-multitenant-without-cert-manager.adoc

```diff
@@ -0,0 +1,187 @@
+
+// Module included in the following assemblies:
+//
+// * service-mesh-docs-main/about/ossm-migrating-migration-guides-assembly.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="migrating-multitenant-without-cert-manager_{context}""]
+== Migrating a multitenant deployment without cert-manager
+
+.Prerequisites
+
+* You have deployed a cluster on {ocp-product-title} 4.14 or later.
+* You are logged in to the {ocp-product-title} web console as a user with the cluster-admin role.
+* You have completed the premigration checklists.
+* You have the {SMProduct} {SMv2Version} Operator installed.
+* You have the {SMProduct} 3 Operator installed.
+* You have `IstioCNI` installed.
+* You have `istioctl` installed.
+* You are not using the cert-manager tool.
```

Comment:
In line with my comments above, I think we should assume that they are not using cert-manager here and leave this off.